tests/test_phase_psf.py

```diff
@@ -449,6 +464,8 @@ def test_ne():
             galsim.AtmosphericScreen(10.0, rng=rng, L0=10.0),
             galsim.AtmosphericScreen(10.0, rng=rng, vx=10.0),
             ]
+    for obj in objs:
+        obj.instantiate()
```

Comment:
They come out as different with regard to `__eq__`, but not `__hash__` so `all_obj_diff` fails.  So I added a kwarg to `all_obj_diff`: `check_hash=True` which can be overridden to False if one wants to test equality but not equal hashes.  I updated the test here to use all_obj_diff with `check_hash=True` on the original objects, and then append an instantiated but otherwise identical object and test again with `check_hash=False`.